docs/platforms/ruby/common/configuration/options.mdx

```diff
@@ -77,193 +141,191 @@ config.breadcrumbs_logger = [:active_support_logger]
+<ConfigKey name="trusted-proxies">
+
+These trusted proxies will be skipped when the SDK computing the user's ip address. `sentry-rails` will automatically inject the value of `Rails.application.config.action_dispatch.trusted_proxies` to this option.
 
 ```ruby
-config.max_breadcrumbs = 30
+config.trusted_proxies = ["2.2.2.2"]
 ```
 
-`trace_propagation_targets`
+</ConfigKey>
 
-An optional property that controls which downstream services receive tracing data, in the form of a `sentry-trace` and a `baggage` header attached to any outgoing HTTP requests.
+## Tracing Options
 
-The option may contain an array of strings or regex against which the URLs of outgoing requests are matched.
-If one of the entries in the list matches the URL of an outgoing request, trace headers will be attached to that request.
-String entries do not have to be full matches, meaning the URL of a request is matched when it _contains_ a string provided through the option.
+<ConfigKey name="traces-sample-rate">
 
-By default, trace headers are attached to every outgoing request from the instrumented client.
+A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app. Either this or <PlatformIdentifier name="traces-sampler" /> must be defined to enable tracing.
 
-`propagate_traces`
+If <PlatformIdentifier name="traces-sample-rate" /> is 0, this means that no new traces will be created. However, if you have another service (for example a JS frontend) that makes requests to your service that include trace information, those traces will be continued and thus transactions will be sent to Sentry.
 
-By default, Sentry injects `sentry-trace` and `baggage` headers to outgoing requests made with `Net::HTTP` to connect traces between services. You can disable this behavior with
+If you want to disable all tracing you need to set <PlatformIdentifier name="traces-sample-rate" />`=None` or set <PlatformIdentifier name="enable-tracing" />`=False`. In this case, no new traces will be started and no incoming traces will be continued.
 
-```ruby
-config.propagate_traces = false
-```
+<PlatformContent includePath="performance/traces-sample-rate" />
 
-`release`
+</ConfigKey>
 
-Track the version of your application in Sentry.
+<ConfigKey name="traces-sampler">
 
-We guess the release intelligently in the following order of preference:
+A lambda or proc responsible for determining the percentage chance a given transaction will be sent to Sentry. It will automatically be passed information about the transaction and the context in which it's being created, and must return a number between `0` (0% chance of being sent) and `1` (100% chance of being sent). Can also be used for filtering transactions, by returning 0 for those that are unwanted. Either this or <PlatformIdentifier name="traces-sample-rate" /> must be defined to enable tracing.
```

Comment:
```suggestion
A lambda or proc that's responsible for determining the chance that a given transaction has of being sent to Sentry (from 0-100%). It will automatically be passed information about the transaction and the context in which it's being created, and must return a number between `0` (0% chance of being sent) and `1` (100% chance of being sent). 

It can also be used for filtering transactions, by returning 0 for those that are of no interest. Either this or <PlatformIdentifier name="traces-sample-rate" /> must be defined to enable tracing.
```